packages/orbit-components/src/Accordion/AccordionSection/index.tsx

```diff
@@ -29,7 +28,10 @@ const AccordionSection = ({
   const [{ height }, ref] = useBoundingRect<HTMLDivElement>({ height: isExpanded ? null : 0 });
 
   return (
-    <AccordionWrapper dataTest={dataTest}>
+    <div
+      className="border-elevation-flat-border-color rounded-normal duration-fast my-xs bg-elevation-flat relative w-full border border-solid ease-in-out"
```

Comment:
The `duration-fast` and `ease-in-out` were here because of the transition. Since the transition was removed, they can also be removed

Reply:
Sorry for the sloppiness here and thank you for being thorough. Removed 👍